### PR TITLE
Variable issuance by on chain contract

### PR DESCRIPTION
**Pull request by alexvandesande**
This EIP proposes a contract that will decide how much and where does block reward goes to.

[Read full proposal](https://github.com/alexvandesande/EIPs/blob/09d943479f1e1c54b36e21da932bd43826ff1a36/EIPS/EIP-1015-on-chain-issuance-control.md)
[Debate on Ethereum Magician's forum](https://ethereum-magicians.org/t/eip-dynamic-block-rewards-with-governance-contract/204)

### CODE REVIEW

File: EIPS/EIP-X-on-chain-issuance-control.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+eip: <to be assigned>
```
Comment (Arachnid) [inline EIPS/EIP-X-on-chain-issuance-control.md]: Please number this 1015 and rename to eip-1015.md.

File: EIPS/EIP-X-on-chain-issuance-control.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+title: Configurable On Chain Issuance
+author: Alex Van de Sande <avsa@ethereum.org>
+discussions-to: <email address>
```
Comment (Arachnid) [inline EIPS/EIP-X-on-chain-issuance-control.md]: Please put the discussion URL here.

File: EIPS/EIP-X-on-chain-issuance-control.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+status: Draft
+type: Core
```
Comment (Arachnid) [inline EIPS/EIP-X-on-chain-issuance-control.md]: This should be "Type: Standards Track"

File: EIPS/EIP-X-on-chain-issuance-control.md (hunk 1 above)
Comment (alexvandesande) [reply EIPS/EIP-X-on-chain-issuance-control.md]: Can I use eip 1000, since it's currently unclaimed 😉

**Comment by siong1987** [comment]
just to relay my opinion from here: https://twitter.com/siong1987/status/987401289582632960

since we aren't sure what the exact implication for this but i like the idea in general, we should instead start doing this for part of the reward(maybe 1/3 of the 3 ETH) instead of the full reward(3 ETH) going into this contract.

**Comment by Arachnid** [comment]
That's 'eip' in lower case. :)

**Pull request merged**